src/test/javascript/spec/component/overview/course-lectures/course-lecture-details.component.spec.ts

```diff
@@ -136,20 +143,22 @@ describe('CourseLectureDetailsComponent', () => {
                 MockProvider(Router),
                 MockProvider(ScienceService),
             ],
-        })
-            .compileComponents()
-            .then(() => {
-                fixture = TestBed.createComponent(CourseLectureDetailsComponent);
-                courseLecturesDetailsComponent = fixture.componentInstance;
-                debugElement = fixture.debugElement;
-
-                // mock profileService
-                profileService = fixture.debugElement.injector.get(ProfileService);
-                getProfileInfoMock = jest.spyOn(profileService, 'getProfileInfo');
-                const profileInfo = { inProduction: false } as ProfileInfo;
-                const profileInfoSubject = new BehaviorSubject<ProfileInfo | null>(profileInfo);
-                getProfileInfoMock.mockReturnValue(profileInfoSubject);
-            });
+        }).compileComponents();
+
+        lectureService = TestBed.inject(LectureService);
+        jest.spyOn(lectureService, 'findWithDetails').mockReturnValue(response);
+        jest.spyOn(lectureService, 'find').mockReturnValue(response);
+
+        fixture = TestBed.createComponent(CourseLectureDetailsComponent);
+        courseLecturesDetailsComponent = fixture.componentInstance;
+        debugElement = fixture.debugElement;
+
+        // mock profileService
```

Comment:
unnecessary in my opinion

Reply:
this was just done for convenience I think. This way to get an CSS element you can omit `fixture.debugElement` every time